www/style/codecheckerviewer.css

```diff
@@ -14,6 +14,10 @@ html, body {
   display: none !important;
 }
 
+.error {
```

Comment:
An error colour has already been introduced once with the product management interface - perhaps it should be revised or reused?

Reply:
I didn't find any definition of this class in the css files. I only found these:
https://github.com/Ericsson/codechecker/blob/d94f08a9fe78e7618b78c818e33000a610336a63/www/style/codecheckerviewer.css#L208
https://github.com/Ericsson/codechecker/blob/d94f08a9fe78e7618b78c818e33000a610336a63/www/style/codecheckerviewer.css#L237

Reply:
Later down in the file there is `mbox mbox-error`. Those are used in the product manager interface for reporting back if e.g. the server rejects your database settings.